App.tsx

```diff
@@ -76,21 +107,54 @@ class App extends Component<{}, appState> {
     });
   };
 
+  success = (location: any) => {
+    const { position } = this.state;
+    position.setLatitude(location.coords.latitude);
+    position.setLongitude(location.coords.longitude);
+    this.setMapLocation(position);
+  };
+
+  failure = (err: any) => {
+    console.log(err);
+  };
+
+  displayCurrentLocation() {
+    const options = {
+      enableHighAccuracy: true,
+      timeOut: 20000,
+      maximumAge: 60 * 60 * 24,
+    };
+    navigator.geolocation.getCurrentPosition(this.success, this.failure, options);
```

Comment:
instead of having 'this.failure' which doesn't do anything, just replace it with null